instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -388,13 +397,31 @@ def _start_response(status, response_headers, *args, **kwargs):
             duration_attrs_old = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.DEFAULT
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_old[HTTP_ROUTE] = wrapped_app_environ.get(
+                    _ENVIRON_REQUEST_ROUTE_KEY
+                )
+                duration_attrs_old[HTTP_TARGET] = wrapped_app_environ.get(
+                    _ENVIRON_REQUEST_ROUTE_KEY
+                )
+
             duration_histogram_old.record(
                 max(round(duration_s * 1000), 0), duration_attrs_old
             )
         if duration_histogram_new:
             duration_attrs_new = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.HTTP
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_new[HTTP_ROUTE] = wrapped_app_environ.get(
+                    _ENVIRON_REQUEST_ROUTE_KEY
+                )
+                duration_attrs_new[HTTP_TARGET] = wrapped_app_environ.get(
+                    _ENVIRON_REQUEST_ROUTE_KEY
+                )
```

Comment:
```suggestion
            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
                duration_attrs_old[HTTP_TARGET] = wrapped_app_environ.get(
                    _ENVIRON_REQUEST_ROUTE_KEY
                )

            duration_histogram_old.record(
                max(round(duration_s * 1000), 0), duration_attrs_old
            )
        if duration_histogram_new:
            duration_attrs_new = otel_wsgi._parse_duration_attrs(
                attributes, _HTTPStabilityMode.HTTP
            )

            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
                duration_attrs_new[HTTP_ROUTE] = wrapped_app_environ.get(
                    _ENVIRON_REQUEST_ROUTE_KEY
                )
```

`target` for old, `route` for new, not both for both.

Reply:
Done!